packages/suite/src/actions/wallet/coinjoinAccountActions.ts

```diff
@@ -164,12 +164,12 @@ const coinjoinSessionStarting = (accountKey: string, isStarting: boolean) =>
         },
     } as const);
 
-const coinjoinSessionAutopause = (accountKey: string, isAutopaused: boolean) =>
+const coinjoinSessionAutostop = (accountKey: string, isAutostoped: boolean) =>
     ({
-        type: COINJOIN.SESSION_AUTOPAUSE,
+        type: COINJOIN.SESSION_AUTOSTOP,
         payload: {
             accountKey,
-            isAutopaused,
+            isAutostoped,
```

Comment:
typo: should be 'isAutostopped'

Reply:
ebac9b502cc404e3706340bcd9f578fdb95590c9

Reply:
926731886a9e40383cdcfa7b9c5e41dc8e4e57d8